test/testdata/order-processing/05_sw.kogito_v1alpha08_kogitoserverlessworkflow_devmode_orderprocessing.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 Red Hat, Inc. and/or its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: sw.kogito.kie.org/v1alpha08
+kind: KogitoServerlessWorkflow
+metadata:
+  name: orderprocessing
+  annotations:
+    sw.kogito.kie.org/description: Workflow for processing Orders and produce Logistics Events
+    sw.kogito.kie.org/version: 1.0.0
+    sw.kogito.kie.org/profile: dev
+spec:
+  resources:
+    configMaps:
+      - configMap:
+           name: orderprocessing-subflows
+  flow:
```

Comment:
Please add `specVersion` and `id` which are required fields in the sdk

Reply:
```suggestion
  flow:
    id: orderprocessing
    specVersion: "0.8"
```

Reply:
https://github.com/serverlessworkflow/sdk-go/blob/main/model/workflow.go#L98
https://github.com/serverlessworkflow/sdk-go/blob/main/model/workflow.go#L71

Reply:
@radtriste they are required, but on our API interface, the ID is the **CR name** and the version is on the annotation. @spolti this needs to be taken care of. On our side, the API should relax these two fields.

I don't think duplicating attributes in the flow is the way to go. What I mean is, the CR name already gives an **id**. And the version of the object should be handled in the annotation.

Reply:
The webhook will add both.
It was is already added here https://github.com/kiegroup/kogito-serverless-operator/blob/main/controllers/workflowdef/json.go#L57

@radtriste what the issue is about?